DSCResources/MSFT_xDatabaseLogin/MSFT_xDatabaseLogin.schema.mof

```diff
@@ -5,7 +5,6 @@ class MSFT_xDatabaseLogin : OMI_BaseResource
     [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
     [Key] String LoginName;
     [Write] String LoginPassword;
-    [Required, ValueMap{"Windows","SQL"}, Values{"Windows","SQL"}] string SqlAuthType;
```

Comment:
You could change it to a write property and achieve most of your goals and not be a breaking change.